lib/jobs/obm-control.js

```diff
@@ -64,18 +64,33 @@ function obmControlJobFactory(
             assert.ok(node, "Node should exist to run OBM command");
             assert.ok(node.obmSettings, "Node should have OBM settings to run OBM command");
 
-            var settings = _.find(node.obmSettings, function(setting) {
+            //var settings = _.find(node.obmSettings, function(setting) {
+            self.settings = _.find(node.obmSettings, function(setting) {
                 return setting ? setting.service === self.obmServiceName : false;
             });
 
-            assert.ok(settings, "Node should have OBM settings for service: " +
+            if(!self.settings){
+                console.log("I'm in the block");
+                logger.info("No settings found that match obmServiceName provided to job.");
+                if(node.obmSettings.length === 1) {
+                    console.log("I'm where I want to be!!!");
+                    self.settings = node.obmSettings[0];
+                    logger.info("Defaulting to obmServiceName "+ self.settings.service +
+                    " for node because only one OBM service present.");
+                }else if(node.obmSettings.length > 1){
```

Comment:
@yyscamper does this case, you brought up, not get handled by the assert found just below, on line 81?